backend/controllers/inventory-controller.js

```diff
@@ -45,6 +45,20 @@ export const getProducts = async (req, res) => {
 
 }
 
+export const getProductById = async (req, res) => {
+    try{
+        const productId = Number(req.params.id)
+        const product = await Prisma.product.findUnique({
```

Comment:
Thank you for covering error catch for find by unique Id.

Anyway, Could you help adjust a bit here? 
Technically, if data on particular is not found in database, our backend uses 404 in most case.

As a rule of thumb for getting data

- List of data -> 200 contains 0 - N records, 500 if any error
- Get detail -> 200 if fully functionality, 404 if not found, 500 if any cause of error